### PR TITLE
Creating page for the next two weeks of events

### PR DESCRIPTION
**Pull request by elanid**

### CODE REVIEW

File: events_retriever.py [hunk 1]
```diff
@@ -29,6 +32,22 @@ def index_page():
     #                 events=contentful_datareader.load_entries(CONTENT_ID_EVENT),
     #                 tags=contentful_datareader.load_entries(CONTENT_ID_TAG))
 
+@app.route('/next2weeks.html')
```
Comment (AVatch) [inline events_retriever.py]: number in the file name - gross :P

File: events_retriever.py [hunk 2]
```diff
@@ -29,6 +32,22 @@ def index_page():
...
+def next_up():
+
+    def naive(d):
```
Comment (AVatch) [inline events_retriever.py]: where is this being used?

File: static/css/2weeks.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
+.main-content-container {
```
Comment (AVatch) [inline static/css/2weeks.css]: maybe call this next2weeks.css so it is consistent with the template name

File: events_retriever.py (hunk 1 above)
Comment (elanid) [reply events_retriever.py]: yeah :p is there something better tho?

**Pull request merged**